tests/ops/service/privacy_request/test_request_service.py

```diff
@@ -0,0 +1,112 @@
+import pytest
```

Comment:
the test coverage here looks pretty good, just noting (as was discussed offline) that ideally we'd like to get some coverage of the polling functionality with the privacy request changing status during the test, to mimic what would actually happen in a real polling use case.

we're hoping to get coverage for that with the broader fides connector tests - if we can't do that, we may want to circle back and look at how we could get that coverage here. for now though, should be fine.